src/visiomode/database/__init__.py

```diff
@@ -0,0 +1,41 @@
+#  This file is part of visiomode.
+#  Copyright (c) 2024 Olivier Delree <odelree@ed.ac.uk>
+#  Distributed under the terms of the MIT Licence.
+
+import typing
+
+import visiomode.models as models
+from visiomode.database.CorruptedDatabaseError import CorruptedDatabaseError
+from visiomode.database.Database import Database
```

Comment:
Same as above, I find the structuring of the imports a little awkward. You could also use an `__all__` export at the module `_init__.py` (https://docs.python.org/3/tutorial/modules.html#importing-from-a-package), but my personal preference would be to group e.g. errors in a single file and define module-level classes at the module `__init__.py` file. f